### DISCUSSION TITLE
Unable to boot Macintosh SE/30 from BlueSCSIv2

### DISCUSSION THREAD

**Discussion by jmcharro** [BlueSCSI/BlueSCSI-v2#208] [Help]
Hello all.

I am trying to boot my SE/30 with no luck at the moment. I am using the latest firmware, and the log.txt shows the image as initialized. I attach log.txt.

[log.txt](https://github.com/user-attachments/files/17607781/log.txt)

I have also tried to boot from a System 7 hard disk utilities disk, but the Apple HD Setup utility does not find any hard disk.

Any help?

**Comment by erichelgeson** [comment]
Please start by going through every step of the Trouble Shooting guide - https://github.com/BlueSCSI/BlueSCSI-v2/wiki/Troubleshooting#resolving-common-issues - It should help narrow down or resolve most common issues.

**Reply by jmcharro** [reply]
Hello Eric.

This is what I have tried so far, as per the troubleshooting guide:

- SD formatted with "Overwrite".
- Checked compatibility of my Macintosh SE/30.
- Updated to the latest firmware (2024.10.26).
- Powered the BlueSCSI via USB.
- Only one image at a time in the SD. Also, created a image using Disk Jockey to test. File naming is correct.
- Only one SCSI device (the BlueSCSI). 
- No ROM drive present.
- Checked jumpers: TRM_ON (tried also OFF), Target Mode.
- No errors in the log.txt file.
[log.txt](https://github.com/user-attachments/files/17608294/log.txt)

- Not tried a different SD card yet because the card is recognized despite being a class 4, and the image mounts correctly:

```
=== Configured SCSI Devices ===
* ID: 0, BlockSize: 512, Type: Fixed, Quirks: Apple, Size: 1638448kB
 
Initialization complete!
```

When booting, the HDD led comes on several times as if the computer is trying to access, but no luck.

I think I have tried everything, so any clue will really be appreciated.

Best regards.

**Reply by erichelgeson** [reply]
> the HDD led comes on several times as if the computer is trying to access

Do you have an internal spinning disk in the SE/30? If so that would be on ID0 as well. The image create in Disk Jockey - is it blank? If so no OS will boot as there's no OS to boot. We have a bunch of known good images on the usage page to test. If none of that works can you enable Debug and provide that log?

**Reply by jmcharro** [reply]
Hello Eric.

No, no internal spinning disk, only the BlueSCSI is connected to the motherboard. I also tried a different SCSI cable just in case. The image from Disk Jockey is blank, I know, but I started with a System 7 floppy disk and the HD utility does not recognize the SCSI device either.

Now the weird thing is that bluescsi.ini file is not recognized (maybe that is a clue...). This is the bluescsi.ini file content in the SD card:

`[SCSI]
Debug=1 ; On`

![image](https://github.com/user-attachments/assets/e9c41bd0-1b62-43e4-acf4-f11d412484e1)

**Reply by erichelgeson** [reply]
Can you validate you're not hiding known file extensions? Sometimes the file is named `bluescsi.ini.txt` and macos will hide the known `.txt` (`command option .` will toggle it in finder) - but yes this an odd clue. I would try a different SD card regardless, even though it might not be the cause.

Does any other SCSI device work/seen on this bus? These computers are 30+ years old and caps/corrosion/faults can exist.

Also make sure you have the proper new lines in the ini file:
```ini
[SCSI]
Debug=1
```

**Comment by jmcharro** [comment]
I will need some weeks to continue the testing. I will get back to you. Thanks a lot for your support.
